src/parser/jobs/whm/index.tsx

```diff
@@ -64,5 +64,10 @@ export const WHITE_MAGE = new Meta({
 			Changes: () => <>Marked WHM as supported for 6.08</>,
 			contributors: [CONTRIBUTORS.INNI],
 		},
+		{
+			date: new Date('2022-04-18'),
+			Changes: () => <>Changed Lily timer and content for 6.1 with misery update</>,
```

Comment:
Nit; consider something more like "Migrate Lily gauge to core gauge and update for 6.1 Misery analysis.", it reads a bit easier and is clearer what you actually changed.